awsconfigs/apps/centraldashboard/kustomization.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ../../../upstream/apps/centraldashboard/upstream/overlays/istio
```

Comment:
wrong location, this should be

apps/centraldashboard/upstream/overlays/kserve. see https://github.com/awslabs/kubeflow-manifests/blob/main/deployments/vanilla/kustomization.yaml#L32

do we need to regenerate the charts?

Reply:
this is the right location.  resource here is point to istio https://github.com/kubeflow/manifests/blob/master/apps/centraldashboard/upstream/overlays/kserve/kustomization.yaml